spec/extensions/auto_validations_spec.rb

```diff
@@ -2,7 +2,15 @@
 
 describe "Sequel::Plugins::AutoValidations" do
   before do
-    db = Sequel.mock(:fetch=>proc{|sql| sql =~ /'a{51}'|'uniq'/ ? {:v=>0} : {:v=>1}})
+    fetch_proc = proc do |sql|
+      if sql =~ /'a{51}'|'uniq'/
+        sql =~ /1 AS one/ ? {:v=>nil} : {:v=>0}
```

Comment:
This is not a style change. The code was unwrapped because new check was added here.